YIF_Backend/Startup.cs

```diff
@@ -228,10 +228,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             #endregion
 
             #region Seeder
-            //SeederDB.SeedData(app.ApplicationServices);
+            SeederDB.SeedData(app.ApplicationServices);
```

Comment:
you didn't comment it, instead of that you commented Seeder at if-state

Reply:
Fixed